src/tools/kata-ctl/src/check.rs

```diff
@@ -51,6 +41,7 @@ pub fn get_single_cpu_info(cpu_info_file: &str, substring: &str) -> Result<Strin
 
 // get_cpu_flags returns a string of cpu flags from cpuinfo, passed in
 // as a string
+#[cfg(test)]
```

Comment:
@hbrueckner the same case, this function is not a module test.

Reply:
correct, will change that.